src/prefect/infrastructure/process.py

```diff
@@ -54,6 +55,10 @@ class Process(Infrastructure):
         default=True,
         description="If set, output will be streamed from the process to local standard output.",
     )
+    cwd: Union[str, bytes, None] = Field(
+        default=None,
+        description="If set, the process will open within the specified path as the working directory.",
```

Comment:
```suggestion
        description="If set, the process will open within the specified path as the working directory. Otherwise, a temporary directory will be created.",
```

Reply:
Liked the documentation addition here. Added it. Didn't realize I could have accepted the change you made here. That's pretty cool. Haven't used GitHub for PRs in a minute. :)